.github/workflows/applications.yml

```diff
@@ -10,20 +10,20 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: ['8.10.7']
-        cabal: ['3.8']
+        ghc: ['8.10.7', '9.2', '9.6']
```

Comment:
Should we add `9.4` to the matrix, since IIUC the docker image is being built against ghc 9.4?

Reply:
It's fixed in 9.4.5. So we could add it back to the matrix.